packages/devtools_app/lib/src/app.dart

```diff
@@ -131,6 +131,8 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
     });
 
     releaseNotesController = ReleaseNotesController();
+
+    setGlobal(NotificationService, Notifications.of(context));
```

Comment:
at this point, should all uses of `Notifications.of(context)` be replaced with `notificationService`?

Reply:
refactored and replaced